tests/ignite/conftest.py

```diff
@@ -25,7 +25,7 @@ def getter(name="test"):
         time.sleep(0.5 * lrank)
         os.makedirs(path, exist_ok=True)
         yield path
-        time.sleep(1.0 * lrank + 1.0)
+        time.sleep(1.0 * lrank + 2.0)
```

Comment:
I think a good approach to see if the issue is here or there is something else we can increase the `sleep time` to a big value and see what's going to happen, if it works then we start reducing it to some proper value

Reply:
Increasing the `sleep time` didn't work btw

Reply:
Thanks for inverstigating that @KickItLikeShika ! 
I also tried to repro that locally and everything works OK. Strange thing is about the actual problem here: `Aborted                 (core dumped)`.